scripts/docker/image-runner.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2024 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
can we set up an optional param in this script to set up customized conf.toml? like https://hub.docker.com/_/redis?

Reply:
Cons of propagating `conf.toml` from outside of docker is, adding filepath configurations.(e.g. `WebStaticFilesPath`) Here could be several options:
- Containing multiple presets inside. (My preference)
- Mount `conf.toml` and its dependencies into docker instances manually.
- Overriding a part of default `conf.toml`.

Feel free to leave any opinions here.

Reply:
I added default `conf.toml` for `dockerIM` usage, and I think users can mount any configuration values by manually setting `docker run`.